apps/ng-app/src/app/pages/about.page.ag

```diff
@@ -1,13 +1,13 @@
 <script lang="ts">
-  import { RouteMeta } from '@analogjs/router';
-  import Hello from '../hello.analog' with { analog: 'imports' };
+import { RouteMeta } from '@analogjs/router';
```

Comment:
I can undo these formatting changes in a bunch of these files. Not sure if that was the Analog VS Code extension that did that. 🤔